src/main/webapp/signup.js

```diff
@@ -158,7 +167,19 @@ function createSignupBody() {
   return {
     gitHubAuthToken: gitHubAuthorizer.getToken(),
     interestTags,
+    emailAddress,
   };
 }
 
+/**
+ * Validates an email syntactically against a regular expression.
+ * Sourced from:
+ * https://stackoverflow.com/questions/46155/how-to-validate-an-email-address-in-javascript
+ * @param {string} email
+ */
+function validateEmail(email) {
+  const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(email).toLowerCase());
+}
```

Comment:
Do we need this?

Reply:
No we don't. I did some further research and it appears that trying to validate emails with regex is generally considered bad practice. Enough validation is already happening through the browser by default by including the `type="email"` in the input tag.